CONTRIBUTING.md

```diff
@@ -7,11 +7,16 @@ understand the choices made by the project.
 
 ## Run the APIv1 test (Python)
 
+:warning: Make sure to follow first the installation instructions in the README!
+
 Tests rely on the Elasticsearch service so first you need to ensure the Elasticsearch container is running.
-You can run the "docker-compose up -d" or you can only run the Elasticsearch container by running "docker-compose start elastic".
+You can run the "docker-compose -f docker-compose.yml.dev up -d" or you can only run the Elasticsearch container by running "docker-compose -f docker-compose.yml.dev start elastic".
+
+(Or see below to use a symlink to the development compose instead instead)
```

Comment:
Similarly, I think the test environment should be isolated from the install. Perhaps we should drop the docker-compose instruction and keep the podman (or docker) command below.